src/w3c/linter-rules/required-sections.js

```diff
@@ -0,0 +1,57 @@
+// @ts-check
+/**
+ * Checks that there is a section that has at least privacy and/or
+ * security and considerations.
+ *
+ * The rule is "privacy" or "security", and "considerations", in any order,
+ * case-insensitive, multi-line check.
+ *
+ */
+
+import { getIntlData, norm, showError } from "../../core/utils.js";
+
+const ruleName = "required-sections";
+export const name = "w3c/linter-rules/required-sections";
+
+const localizationStrings = {
+  en: {
+    msg(sectionTitle) {
+      return `W3C Recommendation track documents require a separate "${sectionTitle}" section.`;
+    },
+    hint(sectionTitle) {
+      return (
+        `Add a \`<section>\` with a "${sectionTitle}" header. ` +
+        "See the [Horizontal review guidelines](https://w3c.github.io/documentreview/#how_to_get_horizontal_review)."
+      );
+    },
+  },
+};
+const l10n = getIntlData(localizationStrings);
+
+function hasSection(headers, expectedText) {
+  const regex = new RegExp(expectedText, "im");
```

Comment:
Maybe we can directly test `norm(textContent).includes(expectedText.toLowercase()`? Why regex?

Reply:
No reason... was trying to basically test the same thing.